readme.md

```diff
@@ -126,6 +126,7 @@ Check out my [blog](https://nikolaskama.me/) and follow me on [Twitter](https://
 * [Go2Shell](http://zipzapmac.com/Go2Shell) - Opens a terminal window to the current directory in Finder.
 * [Cathode](http://www.secretgeometry.com/apps/cathode/) - Vintage terminal for macOS.
 * [Alacritty](https://github.com/jwilm/alacritty) - Cross-platform, GPU-accelerated terminal emulator.
+* [Kitty](https://sw.kovidgoyal.net/kitty/) - Cross-platform, GPU-accelerated terminal emulator.
```

Comment:
Just FYI, kitty is very much not the same thing as alacritty. Just run your eye over the list of major features here: https://sw.kovidgoyal.net/kitty